source/gui/settingsDialogs.py

```diff
@@ -3132,10 +3132,22 @@ def makeSettings(self, settingsSizer: wx.BoxSizer) -> None:
 		index = [x.value for x in AddonsAutomaticUpdate].index(config.conf["addonStore"]["automaticUpdates"])
 		self.automaticUpdatesComboBox.SetSelection(index)
 
+		# Translators: This is the label for a text box in the add-on store settings dialog.
+		addonMetadataMirrorLabel = _("Metadata &mirror")
+		self.addonMetadataMirrorTextbox = sHelper.addLabeledControl(
+			addonMetadataMirrorLabel,
+			wx.TextCtrl,
+		)
+		self.addonMetadataMirrorTextbox.SetValue(config.conf["addonStore"]["baseURL"])
+		self.bindHelpEvent("AddonStoreMetadataMirror", self.addonMetadataMirrorTextbox)
```

Comment:
Generally we don't include the category name in the help event anchor but I think its needed here